### PR TITLE
fix: batch model validation errors

### PR DESCRIPTION
**Pull request by nschonni**
- Remove empty body for RESPONSE_SCHEMA_NOT_IN_SPEC errors
- Wrapped headers for FileGetPropertiesFromNode.json
- leavingpool -> leavingPool
- targetDedicated -> targetDedicatedNodes
- currentDedicated -> currentDedicatedNodes

### CODE REVIEW

**Review by yaohaizh — APPROVED** [review]

**Comment by nschonni** [comment]
@yaohaizh I rebased to see if it would unblock the build status, but it looks like it's not reporting the last job status. At least it's the "test only" so it shouldn't block the merge

**Pull request merged**
